config/routes.rb

```diff
@@ -26,6 +22,12 @@
       end
     end
   end
+  resources :models, only: [:index] do
```

Comment:
This top-level models resource path should probably include the other actions on models (view, edit, etc) rather than keeping them nested inside a library; but it can be done later, it's not urgent.

Reply:
I had the same thought... and the same priority line of thought :)

Reply:
Heh nice, we're thinking very alike here.